static/app/components/events/eventEntry.tsx

```diff
@@ -123,20 +122,18 @@ function EventEntryContent({
       );
 
     case EntryType.SPANS:
-      // XXX: We currently do not show spans in the share view,
-      if (isShare) {
+      // XXX: We currently do not show spans in the share view.
+      if (isShare || !issueTypeConfig?.spanEvidence.enabled) {
         return null;
       }
-      if (issueTypeConfig?.spanEvidence.enabled) {
-        return (
-          <SpanEvidenceSection
-            event={event as EventTransaction}
-            organization={organization as Organization}
-            projectSlug={projectSlug}
-          />
-        );
-      }
-      return <Spans event={event as EventTransaction} />;
+
```

Comment:
Should we be returning `null`? I'm not entirely sure what `issueTypeConfig` is but it seems like we used to return a `<Spans />` component? Maybe we should update the comment if there's an explanation, similar to how we don't show span evidence on sharing.